.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -107,7 +107,7 @@ To help with the load of incoming pull requests:
 
 - [ ] I have reviewed two other [open pull requests][prs] in this repository.
 
-[prs]: https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-desc+review%3Anone
+[prs]: https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-desc+review%3Anone+status%3Asuccess
```

Comment:
Will it also exclude PRs where the CI hasn't started yet as the author is a new contributor?

Reply:
yes it will.

Reply:
```suggestion
[prs]: https://github.com/home-assistant/core/pulls?q=is%3Aopen+is%3Apr+-author%3A%40me+-draft%3Atrue+-label%3Awaiting-for-upstream+sort%3Acreated-desc+review%3Anone++-status%3Afailure
```
To show both success and pending (Just excluding failures)

Reply:
I assume there is no way to include those that only fail due to codecov 🥲 I seem to be generating a lot of those at the moment

Reply:
> Will it also exclude PRs where the CI hasn't started yet as the author is a new contributor?

Ah, good point. I had not thought of that.